app/models/mixins/live_metrics_mixin.rb

```diff
@@ -19,7 +24,7 @@ def first_and_last_capture(interval_name = "realtime")
       end.transpose
       adjust_timestamps(firsts, lasts, interval_name)
     rescue => e
-      _log.error("Hawkular service unavailable: #{e.message}")
+      _log.error("LiveMetrics unavailable: #{e.message}")
```

Comment:
Should this be `Hawkular LiveMetrics`?  Or, will `LiveMetrics` be enough in the logs to know what this is about?